frontends/p4/def_use.h

```diff
@@ -129,7 +130,7 @@ class StructLocation : public WithFieldsLocation {
 /// Interface for locations that support an index operation
 class IndexedLocation : public StorageLocation {
  protected:
-    std::vector<const StorageLocation *> elements;
+    absl::InlinedVector<const StorageLocation *, 8> elements;
```

Comment:
nice, a vector with small-vector optimization will be useful to have. Any reason for the 8 elements in particular?

Reply:
For now it was more or less wild guess just to have the same memory footprint for the object itself.

Reply:
All right, here is the distribution of sizes on switch testcase:
```
     0      1      2      3      4
387633 321734 174704  94070  11821
```

I will check downstream large app as well

Reply:
One downstream app:
```
     0      1      2      3      4     5     6     7     8     9
8932648 6993790 12263 1793251 1684828 1295764 1105095 1213573 394486 34967 
```

Reply:
And sorry, the sizes are for `ProgramPoints` below, not `IndexedLocation`.